### PR TITLE
Add login and create booking pages

### PR DESCRIPTION
**Pull request by rossbickmore**
1) Created bookings controller with new and create actions
2) Created 'new' view

### CODE REVIEW

File: app/controllers/bookings_controller.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
+class BookingsController < ApplicationController
+  before_action :find_flat
```
Comment (Olidem) [inline app/controllers/bookings_controller.rb]: Would call it set_flat

Comment (Olidem) [inline app/controllers/bookings_controller.rb]: Would name it set_flat

File: app/controllers/bookings_controller.rb [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+      redirect_to flat_path(@flat)
+    else
+      render :new
+    end
+    raise
+  end
+
+  private
+
+  def find_flat
```
Comment (Olidem) [inline app/controllers/bookings_controller.rb]: set_flat

File: config/routes.rb [hunk 3]
```diff
@@ -1,4 +1,6 @@
 Rails.application.routes.draw do
+  get 'bookings/create'
+  get 'bookings/new'
```
Comment (Olidem) [inline config/routes.rb]: Please erase.

File: app/controllers/bookings_controller.rb [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    @booking = Booking.create(booking_params)
+    @user = current_user
+    @booking.flat = @flat
+    @booking.user = @user
+    if @booking.save!
+      redirect_to flat_path(@flat)
+    else
+      render :new
+    end
+    raise
```
Comment (Olidem) [inline app/controllers/bookings_controller.rb]: Please erase.

File: app/views/bookings/create.html.erb [hunk 5]
```diff
@@ -0,0 +1,2 @@
+<h1>Bookings#create</h1>
+<p>Find me in app/views/bookings/create.html.erb</p>
```
Comment (Olidem) [inline app/views/bookings/create.html.erb]: Please erase.

**Comment by rossbickmore** [comment]
Completed changes 
:)

**Pull request merged**